internal/reader/ccip.go

```diff
@@ -239,27 +239,17 @@ func (r *CCIPChainReader) MsgsBetweenSeqNums(
 		return nil, err
 	}
 
+	type SendRequestedEvent struct {
+		DestChainSelector cciptypes.ChainSelector
+		Message           cciptypes.Message
+	}
+
 	seq, err := r.contractReaders[sourceChainSelector].QueryKey(
 		ctx,
 		consts.ContractNameOnRamp,
 		query.KeyFilter{
 			Key: consts.EventNameCCIPSendRequested,
 			Expressions: []query.Expression{
-				{
-					Primitive: &primitives.Comparator{
-						Name: consts.EventAttributeSequenceNumber,
-						ValueComparators: []primitives.ValueComparator{
-							{
-								Value:    seqNumRange.Start().String(),
-								Operator: primitives.Gte,
-							},
-							{
-								Value:    seqNumRange.End().String(),
-								Operator: primitives.Lte,
-							},
-						},
-					},
-				},
```

Comment:
If we're going to remove this filter lets do the filtering manually in the for loop below